### ISSUE TITLE
MAJOR - Heading level 1 should be used for the title of the page

### ISSUE THREAD

**Issue by sabderemane** [django/djangoproject.com#1411]
> Major: Certain users are very likely to have difficulty perceiving or operating this.

It visually most prominent heading, ideally not for invisible content ("Django" on homepage). Ideally each page would only have a single h1. ([07:04](https://www.youtube.com/watch?v=dAiMPTOMh1U&t=424s), [SC 1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG22/#info-and-relationships)).

**Comment by specialorange** [comment]
https://github.com/django/djangoproject.com/pull/1419

**Comment by sarahboyce** [comment]
I would propose something like the following:
![docs page with suggested semantic html sections](https://github.com/user-attachments/assets/56055e6b-e77d-473e-90ab-a83841e354c1)

- the top bar is the header `<header>` 
- in the top bar we have navigation `<nav>`
- then it is `<main>` content
- the green "banner" is a `<section>` this will contain a `<h2>` as the information here is often not the page title but rather repeating content (as the title of the selected navigation of the site)
- the docs is a section case where is contains `<search>`
- we then have an `<article> for the docs page
- this has a `<h1>` with the docs title (note that the docs are generated from Sphinx and will always start with a `<h1>` I recommend we don't fight this)
- we already have the side section which has `role="complementary"`, in this we should have some `<nav>` with titles when appropriate

I recommend we roughly adopt this for every page
This means the banner has a h2 and then underneath is the h1
There are 2 pages which might struggle with this currently:
- https://www.djangoproject.com/weblog/
- https://www.djangoproject.com/community/

A quick win would be to add a title to these such as "Recent news" and "Community".
If we want it to look exactly the same, we might be able to have those pages use `<h1>` in the banner but the structure would be slightly inconsistent compared to other pages

**Comment by bmispelon** [comment]
Had to revert the commit, so this is still open

**Comment by bmispelon** [comment]
Fix reapplied in 1ee5166d50806b9da29f64c440c87beec5b6516d
